Sources/SwiftFormatRules/AvoidInitializersForLiterals.swift

```diff
@@ -38,6 +38,16 @@ public final class AvoidInitializersForLiterals: SyntaxFormatRule {
       return super.visit(node)
     }
 
+    guard node.argumentList.count <= 1 else {
```

Comment:
This should probably be `== 1`, because the following is also valid (it initializes to zero):

```swift
let x = Int()
```